src/main/java/org/isf/opd/gui/OpdBrowser.java

```diff
@@ -101,18 +103,21 @@
  *                   - version is now 1.2.2
  * ------------------------------------------
  */
-public class OpdBrowser extends ModalJFrame implements OpdEdit.SurgeryListener, OpdEditExtended.SurgeryListener {
+public class OpdBrowser extends ModalJFrame implements OpdEdit.SurgeryListener, OpdEditExtended.SurgeryListener, PaginationDataProvider<Opd> {
 
 	private static final long serialVersionUID = 2372745781159245861L;
 
 	private static final SimpleDateFormat dateFormat = new SimpleDateFormat("dd/MM/yy");
-
+	private final int defaultPageSize = 100;
+	
 	private JPanel jButtonPanel = null;
 	private JPanel jContainPanel = null;
 	private int pfrmHeight;
 	private JButton jNewButton = null;
 	private JButton jEditButton = null;
 	private JButton jCloseButton = null;
+	private JButton jPreviousPageButton = null;
+	private JButton jNextPageButton = null;
```

Comment:
This two fields that were added appear not to be used.

Reply:
Done, thanks!